doc/source/user_guide/codelist.rst

```diff
@@ -0,0 +1,209 @@
+.. _codelist:
+
+Codelist
+========
+
+A "codelist" (named in reference to code lists in `SDMX <https://sdmx.org/>`_) is a yaml
```

Comment:
A codelist is a list of codes. In this package, the codelists are parsed from yaml files.